packages/intuit/src/http.js

```diff
@@ -0,0 +1,76 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * Options object
+ * @typedef {Object} IntuitOptions
+ * @property {object} query - An object of query parameters to be encoded into the URL
+ * @property {object} headers - An object of all request headers
+ * @property {object} body - The request body (as JSON)
+ * @property {string} [parseAs='json'] - The response format to parse (e.g., 'json', 'text', or 'stream')
+ */
+
+/**
+ * Make a GET request to any intuit endpoint
+ * @example <caption>Get intuit user company information.</caption>
+ * get("/v3/company/9341453908059456/companyinfo/9341453908059456");
```

Comment:
The example needs to read `http.get(...)`. Make sure other examples include the `http` namespace too